ui/src/pages/deployment-details/deployment-details-form/section-example-captures/section-example-captures.tsx

```diff
@@ -45,7 +45,21 @@ export const SectionExampleCaptures = ({
 }: {
   deployment: DeploymentDetails
 }) => {
-  const [files, setFiles] = useState<File[]>([])
+  const [addQueue, setAddQueue] = useState<
```

Comment:
Nice work on adding a queue system for uploads! Perhaps we can use that pattern other places where we have a lot of single requests for bulk change.